### DISCUSSION TITLE
ANN: Non-file tabs moved to a column on the left: Files/New/Log/Find/Settings/Processes are now all on the left

### DISCUSSION THREAD

**Discussion by williamstein** [sagemathinc/cocalc#6404] [Announcements]
I've moved the fixed tabs to buttons on the left, which provides a lot more space for open files, and also helps visually distinguish them from file tabs and project tabs.  This also looks more similar to JupyterLab and VS Code.   That said, the actual functionality is basically identical to before in CoCalc -- clicking on "Files" (say) still displays the file browser, instead of a file panel to the left.

<img width="766" alt="image" src="https://user-images.githubusercontent.com/1276278/216796054-509cfc62-397b-4012-8f29-921a054a0c23.png">

**Comment by DrXyzzy** [comment]
Online docs have been updated include this under "Activity Bar": https://doc.cocalc.com/activity-bar.html
